### PR TITLE
3.0-release-notes

### PR DESCRIPTION
**Pull request by resmaa**
**Release Notes 3.0**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/933#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/00-release-notes.md [hunk 1]
```diff
@@ -14,6 +14,68 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+* Calico Network Policy version 3.24
+* Istio version 1.14.3
+* Image Swap version 1.5.1
+* Generic-VM Libvirt version 1.0.1
+* Generic VM vSphere version1.0.3
+* Tekton-chains version 0.12.0
+* Tekton-operator version 0.61.0
+* K3s_v1.24.4
+* Spectro Proxy version 1.1.0
+* DB services (Mongo, Redis, Postgres)
```
Comment (truegilb) [inline content/docs/00-release-notes.md:65]: Let's call it out PDE only

File: content/docs/00-release-notes.md [hunk 2]
```diff
@@ -14,6 +14,68 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+
+**Packs and Integrations**
+
+* Dex version 2.35.1
+* Harbor version 1.9.3
+* Calico Network Policy version 3.24
+* Istio version 1.14.3
+* Image Swap version 1.5.1
+* Generic-VM Libvirt version 1.0.1
+* Generic VM vSphere version1.0.3
```
Comment (truegilb) [inline content/docs/00-release-notes.md:60]: add a space between version 1.0.3

File: content/docs/00-release-notes.md [hunk 3]
```diff
@@ -14,6 +14,68 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+* Tekton-chains version 0.12.0
+* Tekton-operator version 0.61.0
+* K3s_v1.24.4
```
Comment (truegilb) [inline content/docs/00-release-notes.md:63]: Be consistent "K3s version 1.24.4"

**Review by truegilb — CHANGES_REQUESTED** [review]
Small comment

**Review by truegilb — APPROVED** [review]
Made small changes and approved.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/933#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/933#pullrequestreview-1152663847" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**